src/internal/browser/browser-channel.js

```diff
@@ -247,16 +264,16 @@ function determineWebSocketScheme (config, protocolSupplier) {
 
   if (encryptionOn) {
     // encryption explicitly requested in the config
-    if (!trust || trust === 'TRUST_CUSTOM_CA_SIGNED_CERTIFICATES') {
+    if (!trust || trust === 'TRUST_SYSTEM_CA_SIGNED_CERTIFICATES') {
```

Comment:
For web sockets, no luck of any kind. 

You don't have any control on trust settings, etc. If the browser decides that the server is trusted then you get the connection, otherwise you don't.

This name change, specifically, is because `CUSTOM` word implies that it would be possible to connect if you provide trusted certificates through the config (which is the case for nodejs driver), however it's not so and I believe it's misleading. By naming it as `SYSTEM` now implies that we do not have any control on it.